_posts/2017-12-19-2017-year-in-review.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: post
+title: 'Apache Flink in 2017: Year in Review'
+date: 2017-12-19T09:00:00.000Z
+author: Mike Winters
+excerpt: As 2017 comes to a close, let's take a moment to look back on the Flink community's great work during the past year.
+categories: news
+---
+
+2017 was an exciting year for the Apache Flink® community, and the [release of Flink 1.0 in March](http://flink.apache.org/news/2016/03/08/release-1.0.0.html) marked the first time in Flink's history that the community guaranteed API backward compatibility for all versions in a series. This step forward for Flink was followed by many new and exciting production deployments in organizations of all shapes and sizes, all around the globe.
+
+In this post, we'll look back on the project's progress over the course of 2017, and we'll also preview what 2018 has in store.
+
+{%toc%}
+
+## Community Growth
+
+### Github
+
+First, here's a summary of community statistics from [GitHub](https://github.com/apache/flink). At the time of writing:
+
+-   **Contributors** have increased from 258 in December 2016 to 352 in December 2017 (up **36%**)
+-   **Stars** have increased from 1830 in December 2016 to 3036 in December 2017 (up **65%**)
+-   **Forks** have increased from 1255 in December 2016 to 2070 in December 2017 (up **650%**)
+
+<!-- TODO: How? -->
+
+The community also welcomed **3 new committers in 2016**: Chengxiang Li, Greg Hogan, and Tzu-Li (Gordon) Tai.
```

Comment:
@wints I'm not sure how to find these people…

Reply:
@fhueske might be able to help us put together a list. Otherwise, this info is available on the Flink mailing list, but that would require some manual parsing.